src/action-creators/alert.js

```diff
@@ -1,13 +1,21 @@
+/**
+ * @packageDocumentation
```

Comment:
Why is `@packageDocumention` needed here?

Reply:
> Mark the comment block as @packageDocumentation to let typedoc know that this is documentation for the file (Module) itself 

I removed all `@packageDocumentation` tags from action-creators for testing and typedoc mixed almost all files together.
![Screenshot from 2020-06-17 21-55-17](https://user-images.githubusercontent.com/32064251/84914011-52b7d480-b0e5-11ea-95a5-101626b8d5cf.png)
It defines expandContextThought, setEditingValue, setInvalidState ... as modules because these files don't import anything.

Reply:
Well, we obviously don't want that. Do you see any other solutions? The point of inline documentation is to automate as much as possible from JSDOC comments and folder structure. It's not great to have to specify something explicitly that could be determined automatically.

Reply:
For inner modules with single functions (e.g. `alert`) I kind of like it showing up on the main module page, but including all the internal variables and functions is unacceptable. We probably need the dedicated module page for each inner module to accommodate internal variables.

Reply:
I would like to clarify what you meant.
Here's how submodules are displayed on the main module page. 

![Screenshot from 2020-06-19 15-26-03](https://user-images.githubusercontent.com/32064251/85113291-3b8df980-b241-11ea-993c-ef8355100061.png)

Internal variables and functions are placed inside submodule

![Screenshot from 2020-06-19 15-27-33](https://user-images.githubusercontent.com/32064251/85113417-71cb7900-b241-11ea-8b92-f374af96eb1e.png)

Is it correct or do you want to show only functions having JSDoc comment without any additional functions/variables?

Reply:
That's correct, thanks!